### ISSUE TITLE
Update README, how to use the pulled constraint

### ISSUE THREAD

**Issue by nlarzon** [policy-hub/policy-hub-cli#24]
Improve the README, how to use the pulled constraint.
What is it that you get?

**Comment by Blokje5** [comment]
Good point. I think documentation is one of the first things we should improve. I'll create a PR this weekend to update the REAME.

To answer your question: the pull command downloads the related Rego policies from the policy path defined in the registry. In this case it downloads the examples from constraint: https://github.com/plexsystems/konstraint/examples.
